testsuite/integration-tests-spring/inmodule/src/test/java/org/jboss/resteasy/test/spring/inmodule/SpringMvcHttpResponseCodesTest.java

```diff
@@ -62,6 +65,13 @@
         war.addAsWebInfResource(SpringMvcHttpResponseCodesTest.class.getPackage(), "springMvcHttpResponseCodes/applicationContext.xml", "applicationContext.xml");
         war.addAsManifestResource(new StringAsset("Dependencies: org.springframework.spring meta-inf\n"), "MANIFEST.MF");
         war.addClass(SpringMvcHttpResponseCodesPerson.class);
+
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(
+            new LoggingPermission("control", ""),
```

Comment:
The permission `new LoggingPermission("control", "")` is not needed in order to make the test pass.